jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -797,7 +797,7 @@ def mos_niriss_parser(app, data_dir, table_viewer_reference_name='table-viewer')
                                                 message=".*SRCTYPE is missing or UNKNOWN*")
                         specs = SpectrumList.read(filtered_hdul, format="JWST x1d multi")
                 except UserWarning as e:
-                    raise KeyError(f"The SRCTYPE keyword in the header of file {fname}"
+                    raise KeyError(f"The SRCTYPE keyword in the header of file {fname} "
                                    "is not populated (expected values: EXTENDED or POINT)") from e
```

Comment:
not really applicable to the fix here, but ran into it while testing mosviz and I don't think it warrants its own PR...